### PR TITLE
Surfacing 'clear_tags' function

### PR DESCRIPTION
**Pull request by beldrew**
Exposes the 'clear_tags' function to allow for its use in scripting.

### CODE REVIEW

**Comment by LiquidityC** [comment]
Why close?

**Comment by beldrew** [comment]
A long boring story of me getting confused. 😛 I'll re-open it as it was the easiest change ever.  The function already existed just had to make it 'visible' to lua.

File: resources/help/mud.md [hunk 1]
```diff
@@ -126,3 +126,6 @@ mud.add_tag("MSDP")
 
 The primary use for this function is to indicate what telnet protocols are
 active for the current mud.
+
```
Comment (LiquidityC) [inline resources/help/mud.md:129]: We need a '##' between these two specs to conform to the "format".

**Review by LiquidityC — CHANGES_REQUESTED** [review]
All the logic looks solid. Just one minor formatting thing to fix.

File: resources/help/mud.md (hunk 1 above)
Comment (beldrew) [reply resources/help/mud.md:129]: Oops, missed that.  Fixed

**Review by LiquidityC — APPROVED** [review]

**Comment by LiquidityC** [comment]
Great. It just needs a `cargo fmt` now and all should go through smooth.

**Pull request merged**